libs/estdlib/src/proplists.erl

```diff
@@ -27,10 +28,147 @@
 %%-----------------------------------------------------------------------------
 -module(proplists).
 
--export([get_value/2, get_value/3]).
+-export([
+    property/1, property/2,
+    compact/1,
+    unfold/1,
+    delete/2,
+    get_bool/2,
+    get_all_values/2,
+    get_value/2, get_value/3,
+    is_defined/2,
+    lookup/2,
+    lookup_all/2,
+    from_map/1,
+    to_map/1
+]).
 
 -type property() :: atom() | {term(), term()}.
 
+% Taken from `otp/blob/master/lib/stdlib/src/proplists.erl`
```

Comment:
Code comment need `' instead of the expected ``... so this should be:
`otp/blob/master/lib/stdlib/src/proplists.erl'

Reply:
same for all of the other occurrences of \`anything\`.

Reply:
I don't think that is an issue, that comment is not processed from doxygen & friends.
It looks like it is building now: https://github.com/atomvm/AtomVM/actions/runs/8885471737/job/24396899109?pr=1144
am I correct?

Reply:
Right. I picked one of the few places that it didn't matter as an example.

Yes, it's building now. Looks like all green lights.